service/history/execution/mutable_state_builder_test.go

```diff
@@ -990,3 +994,274 @@ func (s *mutableStateSuite) buildWorkflowMutableState() *persistence.WorkflowMut
 		VersionHistories:    versionHistories,
 	}
 }
+
+func TestNewMutableStateBuilderWithEventV2(t *testing.T) {
+
```

Comment:
nit: remove empty first lines in functions

Reply:
sure, but I'd suggest putting that in lint if it's a thing we want to do more generally

Reply:
Would be nice to remove generous new lines like this via linter which IMO hurts readability a little bit. Consistently spaced codebase is much easier to read thru. We are far from consistency so this new line is not in the top of my list :) Maybe @Groxx can quickly come up with the magic regex for this too